src/Pim/Component/Connector/Processor/Normalization/VariantGroupProcessor.php

```diff
@@ -102,7 +96,6 @@ protected function fetchMedia(GroupInterface $variantGroup, $directory)
         }
 
         $identifier = $variantGroup->getCode();
-        $this->variantGroupUpdater->update($variantGroup, ['values' => $productTemplate->getValuesData()]);
```

Comment:
This line is cannot work, hence the export failure, and has absolutely no use at all.